src/main.cpp

```diff
@@ -429,8 +431,9 @@ void setup() {
     //  Build shader;
     //
     if (iFrag != -1) {
+        fragPath = &files[iFrag].path;
```

Comment:
You are only allowed to specify a maximum of 1 *.frag file on the command line. `fragPath` is the name of this *.frag file, or it is nullptr if not specified. The `files` array can contain a maximum of 1 fragment file, whose index is the global variable `iFrag` (-1 if there is no frag file). There is no code for watching if an included file has changed, and included files are not listed in the `files` array.

The code you highlighted is not executed in the case of `#include` processing. This is localized inside of `loadFromPath()` in `utils.h`.